pkg/generators/forwarding/fluentd/source.go

```diff
@@ -2,20 +2,25 @@ package fluentd
 
 import (
 	"fmt"
+	"strings"
 
 	logforward "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
 	"github.com/openshift/cluster-logging-operator/pkg/constants"
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
-func (engine *ConfigGenerator) generateSource(sources sets.String) (results []string, err error) {
+func (engine *ConfigGenerator) generateSource(sources sets.String, appNs sets.String) (results []string, err error) {
 	//looking to control order
 	templates := []string{}
+	appNsPaths := []string{}
 	if sources.Has(string(logforward.LogSourceTypeInfra)) {
 		templates = append(templates, "inputSourceJournalTemplate")
 	}
 	if sources.Has(string(logforward.LogSourceTypeApp)) {
 		templates = append(templates, "inputSourceContainerTemplate")
+		for _, ns := range appNs.List() {
+			appNsPaths = append(appNsPaths, fmt.Sprintf("\"/var/log/containers/*_%s_*.log\"", ns))
```

Comment:
Making the magic string `/var/log/containers` a constant helps to prevent accidental mistakes.

Reply:
will update when merging with GA API implementation.